docs/azure-tre-features/airlock.md

```diff
@@ -0,0 +1,130 @@
+# Airlock
+
+In a Trusted Research Environment (TRE) the workspaces represent a security boundary that enables researchers to access data, execute analysis, apply algorithms and collect reports. The airlock capability brings the only mechanisme that allows users to `import` or `export` data and tools in a secure fashion with a manual approval.
+This constitutes the mechanism focused in preventing malware inside TRE and/or TRE workspaces and preventing data exfiltration, while allowing researchers to execute their tasks.
+The airlock feature brings several actions: ingress/egress Mechanism; Data movement; Security gates; Approval mechanism and Notifications. As part of TRE's Safe settings all activity must be tracked for auditiing purpose.
```

Comment:
This sentence is not 100% clear to me.
ingress/egress Mechanism vs Data movement, what the differencce
security gates - malware scanning?

Reply:
Yes, the idea is to enumerate key aspects from Airlock (and thinking in the future):
- ingress/egress Mechanism: is focused on how you move from an internal security boundary (export) to external location. For export the other way around
- Data Movement is more around how data is moved between several locations for different purpose
- Security Gates was the term we found for any (compute) task that may be brought into the process. Curretnly we have malware scanning only. In the future you could have PII scanning and others.

Let me rephrase it to make it clearer.